### DISCUSSION TITLE
feature/plugin question

### DISCUSSION THREAD

**Discussion by an1m4l** [Jyers/Marlin#1090] [Q&A] [answered]
does binary firmware flash work with this? or arc welder/eprom editor?

**Comment by adam3654** [comment, marked as answer]
(Yes)^3

**Reply by an1m4l** [reply]
yes to all 3? awesome

**Reply by Jyers** [reply]
If by binary firmware flash you mean Marlin binary file transfer, no unfortunately as it doesn't work with meatpack. The other two definitely work though!
